Makefile

```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
```

Comment:
Why not use the embedded-cluster-release-builder binary we already have? We could replace it with this one I don't mind.

Reply:
this is a little more specific to the workflow i am trying to crfeate